packages/router/src/lib/helpers/prepare.ts

```diff
@@ -33,39 +35,73 @@ export const validBidExpiry = (bidExpiry: number, currentTime: number) => bidExp
  * Returns the swapRate
  *
  * @param TODO
- * @returns The swapRate, determined by the AMM
+ * @returns The swapRate, determined by the stableSwap
  *
  * @remarks
- * TODO: getSwapRate using AMM
+ * TODO: getSwapRate using stableSwap
  */
-export const getSwapRate = async (): Promise<string> => {
-  return "1";
+export const getSwapRate = async (
```

Comment:
this doesn't necessarily return the swap *rate* but instead returns the swapped amount, right? if so, we should rename the function so that people (*cough* me *cough*) don't get confused

Reply:
right. will rename!.